util/short_client_token.py

```diff
@@ -26,13 +26,26 @@ def adobe_base64_decode(cls, str):
         encoded = str.replace(":", "+").replace(";", "/").replace("@", "=")
         return base64.decodestring(encoded)
 
-    EPOCH = datetime.datetime(1970, 1, 1)
+    # The JWT spec takes January 1 1970 as the epoch.
+    JWT_EPOCH = datetime.datetime(1970, 1, 1)
+
+    # For the sake of shortening tokens, the Short Client Token spec
+    # takes January 1 2017 as the epoch, and measures time in minutes
+    # rather than seconds.
+    SCT_EPOCH = datetime.datetime(2017, 1, 1)
```

Comment:
Apart from these constants, none of the code in this file makes a difference, but I changed it to be consistent with what the circ manager will be generating in the future.